Azure/sonic-buildimage
# [YANG] Enhance the port yang model with new port fields: adv_speeds, interface_type and adv_interface_types

**- Why I did it**
Enhance the port yang model with new port fields: adv_speeds, interface_types and adv_interface_types
Refer to HLD https://github.com/Azure/SONiC/pull/732 

**- How I did it**
Add fields adv_speeds, interface_type, adv_interface_types

**- How to verify it**
Add yang unit test and verify build is not broken.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**


**- A picture of a cute animal (not mandatory but encouraged)**
